### PR TITLE
Replace entryType registration hook with Note

### PR DESCRIPTION
**Pull request by npm1**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/paint-timing/pull/42.html" title="Last updated on Jun 13, 2019, 11:13 PM UTC (6bfff9c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/paint-timing/42/fed4219...6bfff9c.html" title="Last updated on Jun 13, 2019, 11:13 PM UTC (6bfff9c)">Diff</a>

### CODE REVIEW

**Comment by npm1** [comment]
+domenic for thoughts on whether this kind of note is reasonable, and what good replacements for 'should' are

**Comment by domenic** [comment]
I don't understand why we're replacing a normative requirement with a note here.

File: painttiming.bs [hunk 1]
```diff
@@ -80,9 +82,7 @@ The {{PerformancePaintTiming}} interface {#sec-PerformancePaintTiming}
 * The {{PerformanceEntry/startTime}} attribute must return a {{DOMHighResTimeStamp}} of when the paint occured.
 * The {{PerformanceEntry/duration}} attribute must return 0.
 
-A user agent implementing {{PerformancePaintTiming}} must perform the following steps:
-1. Run the <a>register a performance entry type</a> algorithm with <code>"first-paint"</code> as input.
-1. Run the <a>register a performance entry type</a> algorithm with <code>"first-contentful-paint"</code> as input.
+NOTE: A user agent implementing {{PerformancePaintTiming}} is compelled to include <code>"first-paint"</code> and <code>"first-contentful-paint"</code> in {{PerformanceObserver/supportedEntryTypes}} for {{Window}} contexts.
```
Comment (yoavweiss) [inline painttiming.bs]: s/is compelled/would need to/ ?
Maybe also add "so that developers would be able to detect the feature's support" or some such?

**Comment by npm1** [comment]
> I don't understand why we're replacing a normative requirement with a note here.

Because we went with the registry model which you were strongly in favor of, which in particular means that there is no longer a registration hook, so we cannot use normative text. See also https://github.com/w3c/performance-timeline/issues/117#issuecomment-492386799

**Comment by yoavweiss** [comment]
> I don't understand why we're replacing a normative requirement with a note here.

The normative requirement have moved to the PerformanceTimeline spec and to a related registry of entry types. The note should be a reminder for implementers to comply with that requirement (along with related tests in the relevant directory)

**Comment by domenic** [comment]
I remember being in favor of having the spec for entryTypes state what it returns. I don't know how this registry model should work, but it sounds like @yoavweiss has opinions, so I'm happy to let him guide here.

**Pull request merged**